packages/api/src/schema/schema.yaml

```diff
@@ -814,6 +818,13 @@ components:
               type: string
             extraData:
               type: string
+        webhookId:
```

Comment:
It would be nice if this was a `oneOf` as well. Even better than the gate payload since it is actual user input. We shouldn't allow users to specify a `webhookId` if their type is `jwt`, nor the lit stuff when the type is webhook, etc etc.

As an example of `oneOf` check the `asset.source`, it's pretty much the same use of that: https://github.com/livepeer/livepeer-com/blob/62b2672ee600f55fe5df5c2bf5a2532ef10963ed/packages/api/src/schema/schema.yaml#L1095